datasets/modeled/transport/dimensions/subsectors.csv

```diff
@@ -0,0 +1,2 @@
+id
+bev
```

Comment:
This needs to include one row for each subsector in your data. Also, each dimension record file has to have a `name` column.

Reply:
sorry, committed too early - this is WIP

Reply:
fixed now, see new commit